app/src/main/java/com/sun/efood/utils/FragmentUtil.kt

```diff
@@ -0,0 +1,50 @@
+package com.sun.efood.utils
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentTransaction
+
+object FragmentUtil {
+
+    fun addFragment(
+        fragmentManager: FragmentManager,
```

Comment:
Em chuyển sang viết dưới dạng Extension 
Extension sẽ gen cho em 1 class tương tự class Util của em trong Java 
Chú ý viết ở top-level, không viết trong object hay class 
```suggestion
fun FragmentManager.addFragment(
```